tests/e2e/cucumber/steps/api.ts

```diff
@@ -267,12 +268,13 @@ Given(
   ): Promise<void> {
     const user = this.usersEnvironment.getUser({ key: stepUser })
     for (const info of stepTable.hashes()) {
-      await api.dav.addMembersToTheProjectSpace({
+      await api.share.addMembersToProjectSpaceUsingSharingNG({
```

Comment:
let's use existing name
```suggestion
      await api.share.addMembersToTheProjectSpace({
```